src/pages/home/report/ContextMenu/ContextMenuActions.js

```diff
@@ -204,14 +204,39 @@ export default [
                     Clipboard.setString(modifyExpenseMessage);
                 } else if (ReportActionsUtils.isMoneyRequestAction(reportAction)) {
                     const originalMessage = _.get(reportAction, 'originalMessage', {});
-                    const transaction = TransactionUtils.getTransaction(originalMessage.IOUTransactionID);
-                    const {amount, currency, comment} = ReportUtils.getTransactionDetails(transaction);
-                    const formattedAmount = CurrencyUtils.convertToDisplayString(amount, currency);
-                    const displaymessage = Localize.translateLocal('iou.requestedAmount', {
-                        formattedAmount,
-                        comment,
-                    });
-                    Clipboard.setString(displaymessage);
+                    let displayMessage;
+                    if (originalMessage.type === CONST.IOU.REPORT_ACTION_TYPE.PAY) {
```

Comment:
i think we can move all logic from if to some util function and return here just displayMessage
getTaskReportActionMessage for reference